frontend/src/pages/Login.vue

```diff
@@ -112,6 +113,7 @@ const onSubmit = handleSubmit(async (values) => {
     })
     const { login: loginResponse } = result.data
     await store.dispatch('login', loginResponse)
+    store.commit('gmsActive', CONFIG.GMS_ACTIVE)
```

Comment:
Why you store the config value in the local store of the browser?
So when the user was still logged in, while we rebuild the frontend with disabled gms, the users frontend will still try to render the gms card and ask the backend for gms login key? But the backend will throw an error, if it was disabled.

Reply:
Ok, yes that's right.
The previous used control-attribut "gmsAllowed" still exists in the store, but this is a user-dependend value and not equal the global config setting. That's the reason, why i tried to add the "gmsActive" in the store.

But you are right, i will change and remove it frome the store and read the control-attribute directly from config instead of the store